packages/react/utils/src/useCallbackRef.tsx

```diff
@@ -1,11 +1,10 @@
 import * as React from 'react';
 
 /**
- * A custom hook that converts a callback to a ref
- * to avoid triggering re-renders when passed as a prop
- * or avoid re-executing effects when passed as a dependency
+ * A custom hook that converts a callback to a ref to avoid triggering re-renders when passed as a
+ * prop or avoid re-executing effects when passed as a dependency
  */
-export function useCallbackRef<T extends (...args: any[]) => any>(callback?: T): T {
+export function useCallbackRef<T extends (...args: any[]) => any>(callback: T | undefined): T {
```

Comment:
I think this typing makes more sense, as we should always expect an argument here but it might be undefined.